ERCS/erc-intent-7702.md

```diff
@@ -0,0 +1,159 @@
+---
+eip: tbd
```

Comment:
```suggestion
eip: 7806
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
👍 Updated